source/features/releases-tab.tsx

```diff
@@ -103,11 +103,16 @@ async function init(): Promise<false | void> {
 			releasesTab.setAttribute('aria-current', 'page');
 		}
 
-		select('.dropdown-divider', repoNavigationBar)!.before(
-			createDropdownItem('Releases', `/${repoUrl}/releases`, {
-				'data-menu-item': 'rgh-releases-item'
-			})
-		);
+		const moreDropdown = select('.dropdown-divider', repoNavigationBar);
+
+		if (moreDropdown) {
+			releasesTab.classList.add('hx_underlinenav-item', 'js-responsive-underlinenav-item');
```

Comment:
If this change is not done it errors out since github is looking to hide but it cant since it does not exist